web/packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -175,4 +175,8 @@ export default class StoreUserContext extends Store<UserContext> {
   hasPluginsAccess() {
     return this.state.acl.plugins.list || this.state.acl.plugins.create;
   }
+
+  getDeviceTrustAccess() {
+    return this.state.acl.deviceTrust.list;
```

Comment:
`getXXXAccess` implies we want to get the entire access object
```suggestion
    return this.state.acl.deviceTrust;
```